jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/SamDebugSupport.kt

```diff
@@ -37,7 +38,7 @@ interface SamDebugSupport {
         debugPorts: List<Int>
     ): XDebugProcessStarter?
 
-    fun isSupported(): Boolean = true
```

Comment:
Maybe add a comment that this will only be Python/Java? it looks kind of weird because it takes the argument then does nothing with it

Reply:
Added comment